src/cli_subcommands.jsonc

```diff
@@ -0,0 +1,38 @@
+// Configuration for the top-level subcommands offered by the mdtf.py driver
+// script. This is read by MDTFSubcommandDispatch. 
+//
+// DO NOT modify this file. Instead, run 'mdtf install', which will generate a
+// copy in /sites which you can edit.
+//
+// All text to the right of an unquoted "//" is a comment and ignored, as well
+// as blank lines (JSONC quasi-standard.)
+{
+  "title": "COMMANDS", 
+  "description": "Subcommand functionality. Use '%(prog)s <command> --help' to get help on options specific to each command.",
+  "help": null,
+  "metavar": "<command> is one of:",
+  "subcommands": {
+    // "install": {
```

Comment:
Are the commented-out lines TODO features?

Reply:
Yes -- the intention was to have all the separate features of the package as subcommands of a single script, rather than different scripts (because scripts share framework code, and need to run in the `_MDTF_base` conda environment [except for the installer script]). Because this isn't backwards compatible, it will need to wait for the next release.
Commented-out lines are removed in NOAA-GFDL/MDTF-diagnostics@991995d2746ecb8b54fea8145d2fab3e96eaa83e.